pkgs/development/python-modules/pytest-testmon/default.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs
+, lib
+, pythonPackages
```

Comment:
```suggestion
, pytest
, coverage
```

You need to add them a function arguments like this

Reply:
I'm glad you didn't mention it before. I just figured it out independently. I love when there's no magic.